client/js/lib/d3.js

```diff
@@ -136,15 +137,18 @@ module.exports = function ({
 		.attr('stop-opacity', 1);
 
 	const rootNode = svg.select('.rootNode');
+	const baseColor = ringColor ||  "#fff1e0";
+	debugger;
+	const dampener = 1.1;
 	for (let i=0; i<rings.length; i++) {
 		rootNode.append('circle')
 			.attr('class', 'background')
 			.attr('r', (rings[i].max + 1) * ringSize)
-			.style('fill', rings[i].fill);
+			.style('fill', baseColor);
 		rootNode.append('circle')
 			.attr('class', 'background')
 			.attr('r', (rings[i].max + 1 ) * ringSize)
-			.style('fill', `url(#radgrad)`);
+			.style('fill', `rgba(0,0,0, 0.${ (i * dampener) | 0})`);
```

Comment:
Since dampener is constant and unmutable please just change the base color instead so there are less dom elements.

I only did it this way before because you can't define a gradient background in svg